rhobs/alerting/data_plane/prometheus.gitops_sync_alerts.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-gitops-sync-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: gitops_sync_alerts
+    interval: 1m
+    rules:
+    - alert: GitOpsApplicationSyncErrors
+      expr: |
+        sum(argocd_app_info{sync_status="Synced", namespace="gitops-service-argocd"})
+        /
+        sum(argocd_app_info{namespace="gitops-service-argocd"}) < 0.95
+      for: 5m
+      labels:
+        severity: warning
+        slo: true
+      annotations:
+        summary: >-
+          Less than 95% of GitOps applications are in synced state
+        description: >-
+          The percentage total of all Argo CD applications that are in Synced state is less than 95%.
+        runbook_url: https://gitlab.cee.redhat.com/rhtap/docs/sop/-/blob/main/gitops/deploy-from-git-to-k8s.md
+    - alert: GitOpsApplicationNoSyncApps
+      expr: |
+        (
+          sum(argocd_app_info{sync_status="OutOfSync", namespace="gitops-service-argocd"}) +
```

Comment:
We will get no answer if we have no applications with `sync_status="OutOfSync"` or `sync_status="Unknown"`
We will get  `No datapoints found`